guide/interactions/buttons.md

```diff
@@ -14,22 +14,22 @@ Buttons are part of the `MessageComponent` class, which can be sent via messages
 You can have a maximum of five `ActionRow`s per message, and five buttons within an `ActionRow`.
 :::
 
-To create a button, use the `MessageActionRow()` and `MessageButton()` builder functions and then pass the resulting object to `CommandInteraction#reply()` as `InteractionReplyOptions`:
+To create a button, use the `ActionRowBuilder()` and `ButtonBuilder()` functions and then pass the resulting object to `CommandInteraction#reply()` as `InteractionReplyOptions`:
 
 ```js {1,7-13,15}
-const { MessageActionRow, MessageButton } = require('discord.js');
+const { ActionRowBuilder, ButtonBuilder, ButtonStyle } = require('discord.js');
 
 client.on('interactionCreate', async interaction => {
-	if (!interaction.isCommand()) return;
+	if (!interaction.isChatInputCommand()) return;
```

Comment:
I find it odd that the above text states to use `CommandInteraction#reply()` yet we are only allowing chat input interactions through. This will work with context menus too.